components/TextFields/src/ContainedInputView/private/MDCTextControlStyleFilled.m

```diff
@@ -158,7 +166,9 @@ - (void)applyStyleToView:(UIView *)view
   UIBezierPath *filledSublayerBezier = [self filledSublayerPathWithTextFieldBounds:view.bounds
                                                                    containerHeight:containerHeight];
   self.filledSublayer.path = filledSublayerBezier.CGPath;
-  if (self.filledSublayer.superlayer != view.layer) {
+
+  BOOL isNotYetInTextControlLayerHierarchy = self.filledSublayer.superlayer != view.layer;
```

Comment:
Is is sufficient to check that the filledSublayer has a superlayer?
change the check to (self.filledSublayer.superlayer != nil). Then it is self explanatory?

I only ask because the bool is a mouthful.

My suggestion only works if the the filledSublayer is only added t one layer and not any other.

Reply:
As it stands right now, if filledSublayer has a superlayer it's only ever going to be the textControl's layer. But if that were to change we'd have to change it back to `self.filledSublayer.superlayer != view.layer`. I think checking that the style is currently applied to the right view, as opposed to any view, is a good thing to do. Should I shorten the local variable name though? I've never minded really descriptive variable names because they sometimes prevent ambiguity but I can understand why they can also be seen as a mouthful.